src/python/ensembl/utils/__init__.py

```diff
@@ -12,5 +12,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Utils module"""
-
-from .rloader import *
```

Comment:
you just broke the handover deployment while doing that.

Reply:
No... in fact your didn't but you could have :-) 
- https://github.com/Ensembl/ensembl-prodinf-datachecks/blob/bbca777d04b9d2dac99acdb8b6a3c3efef75d641/src/ensembl/production/datacheck/config.py#L22
- https://github.com/Ensembl/ensembl-prodinf-handover/blob/1d384cd212d3107e0c1af850caf025486c1c70db/src/ensembl/production/handover/config.py#L22

Reply:
Phew! Will be more cautious in the future for sure! :eyes: